src/components/rooms/FAQ/FAQRoom.tsx

```diff
@@ -32,34 +32,44 @@ const questions = [
     {
       question: "Will there be other activities besides hacking?",
       answer:
-        "Of course! We'll have workshops and other activities throughout the event."
+        "Of course! We'll have workshops you can learn from, sponsored events you can network at, virtual game nights where you can meet other hackers, and so much more! You can find the complete schedule on our website."
     },
     {
       question: "Can I submit my older projects?",
-      answer: "You are free to work on any project you like, however we require that all devpost submissions be projects that were made during the hackathon and uniquely for the hackathon."
+      answer: "You are free to work on any project you like; however, we require that all Devpost submissions be projects that were started and made during our hackathon and uniquely for VandyHacks VII."
     },
     {
       question: "What does a virtual hackathon mean?",
       answer:
         "An entirely remote hackathon encompasses everything that an in-person hackathon has, except now you can do it from the comfort of your own bedroom! This includes a Discord server to meet other hackers, pre-recorded workshops, virtual game nights, and a live stream of the closing ceremony."
     },
+    {
+      question: "Where can I find the Discord link?",
+      answer:
+        "Once we've approved your application, we'll send you an email with the invitation to the Discord. The invitation may end up in your spam folder - if it's still not there, feel free to send us an email at info@vandyhacks.org!"
+    },
     {
       question: "Who can apply?",
       answer:
-        "Anyone who is 18 years or older (we will be checking ID) and is currently enrolled in college or university with a valid student ID. This means both international and non-Vandy students are welcome, as well!"
+        "Anyone who is 18 years or older and is currently enrolled in college or university with a valid student ID. This means both international and non-Vandy students are welcome, as well!"
     },
     {
       question: "I have no coding experience. Can I still attend?",
       answer:
         "Even more reason for you to come! We will be hosting beginners' workshops for you to get started, and our mentors can help you out along the way. We greatly encourage new hackers to attend, and no prior experience is necessary!"
     },
+    {
+      question: "I'm currently residing outside of the U.S. Can I still attend? ",
+      answer:
+        "Yes, you can still attend! Due to shipping restrictions, if you reside outside the U.S., we will be unable to send you any swag items or prizes. However, other than that, you are fully able to participate in all of the hacking and the other events we will be holding!"
```

Comment:
```suggestion
        "Yes, you can still attend! Due to shipping restrictions, if you reside outside the U.S., we will be unable to send you any swag items or physical prizes. However, other than that, you are fully able to participate in all of the hacking and the other events we will be holding!"
```
I assume virtual prizes like gift cards et al. should be okay for international?